### PR TITLE
Add Contact, Account, ContactAccount Property Resolvers and Resource Loaders

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no yes
| BC breaks? | no
| Deprecations? | no yes 
| Fixed tickets | fixes # 
| Related issues/PRs | # 
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Add Contact and Account Property Resolvers.

#### Why?

See https://github.com/sulu/sulu/pull/7611 and https://github.com/sulu/SuluContentBundle/pull/271

### CODE REVIEW

File: src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/PropertyResolver/AccountSelectionPropertyResolver.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+use Sulu\Bundle\ContentBundle\Content\Application\ContentResolver\Value\ContentView;
+use Sulu\Bundle\ContentBundle\Content\Application\PropertyResolver\PropertyResolverInterface;
+
+class AccountSelectionPropertyResolver implements PropertyResolverInterface
+{
+    public function resolve(mixed $data, string $locale, array $params = []): ContentView
+    {
+        if (!\is_array($data)
+            || 0 === \count($data)
+            || !\array_is_list($data)
```
Comment (Prokyonn) [inline src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/PropertyResolver/AccountSelectionPropertyResolver.php:26]: do we really care about the keys ? 🤔

File: src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/ResourceLoader/AccountResourceLoader.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class AccountResourceLoader implements ResourceLoaderInterface
+{
+    public const RESOURCE_LOADER_KEY = 'account';
+
+    /**
+     * @param ContactManagerInterface<AccountInterface, AccountApi, AccountAddressEntity> $accountManager
+     */
+    public function __construct(
+        private ContactManagerInterface $accountManager,
```
Comment (Prokyonn) [inline src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/ResourceLoader/AccountResourceLoader.php:30]:
```suggestion
        private AccountManagerInterface $accountManager,
```

**Review by Prokyonn — CHANGES_REQUESTED** [review]

File: src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/PropertyResolver/AccountSelectionPropertyResolver.php (hunk 1 above)
Comment (alexander-schranz) [reply src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/PropertyResolver/AccountSelectionPropertyResolver.php:26]: Test for smart content catches this.

File: src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/ResourceLoader/AccountResourceLoader.php (hunk 2 above)
Comment (alexander-schranz) [reply src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/ResourceLoader/AccountResourceLoader.php:30]: There is no accountmanagerinterface contact and account manager use the same interface

Comment (Prokyonn) [reply src/Sulu/Bundle/ContactBundle/Infrastructure/Sulu/Content/ResourceLoader/AccountResourceLoader.php:30]: 🙈

**Review by Prokyonn — APPROVED** [review]

**Pull request merged**